Gems/PhysX/Code/Source/EditorHeightfieldColliderComponent.cpp

```diff
@@ -61,15 +65,49 @@ namespace PhysX
                         "Configuration of the collider")
                         ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
                         ->Attribute(AZ::Edit::Attributes::ChangeNotify, &EditorHeightfieldColliderComponent::OnConfigurationChanged)
+
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default, &EditorHeightfieldColliderComponent::m_useBakedHeightfield, "Use Baked Heightfield",
+                        "Selects between a dynamically generated heightfield or a prebaked one. Note: This affects only game time heightfields")
```

Comment:
Is it true that it only affects the heightfield at game time? If so, should we change it to affect things at edit time too, so that the debug wireframe shows the actual heightfield that will be used?